flex/storages/rt_mutable_graph/loader/abstract_arrow_fragment_loader.cc

```diff
@@ -38,6 +38,53 @@ bool check_primary_key_type(std::shared_ptr<arrow::DataType> data_type) {
   }
 }
 
+void set_column_from_string_array_safe(
```

Comment:
Is it possible to reuse the function `set_column_from_string_array`, but passing a indicator to indicate which set method should be used?